thinc/layers/pytorchwrapper.py

```diff
@@ -34,6 +34,8 @@ def PyTorchWrapper(
     pytorch_model,
     convert_inputs: Optional[Callable] = None,
     convert_outputs: Optional[Callable] = None,
+    mixed_precision: bool = False,
+    grad_scaler: Optional[PyTorchGradScaler] = None,
```

Comment:
We need to increment the versioning here to `PyTorchWrapper.v2` to make sure it's clear that these two parameters are only supported in the newer version and from a specific Thinc release. Both definitions can be here in the same file though.

Reply:
Ah, right, should be fixed now.